lib/matplotlib/rcsetup.py

```diff
@@ -767,6 +759,17 @@ def validate_cycler(s):
     else:
         raise ValueError("object was not a string or Cycler instance: %s" % s)
 
+    unknowns = cycler_inst.keys - (set(_prop_validators.keys()) |
+                                   set(_prop_aliases.keys()))
+    if unknowns:
+        raise ValueError("Unknown artist properties: %s" % unknowns)
+
+    # Not a full validation, but it'll at least normalize property names
+    # A fuller validation would require v0.10 of cycler.
+    for prop in cycler_inst.keys:
+        norm_prop = _prop_aliases.get(prop, prop)
+        cycler_inst.change_key(prop, norm_prop)
```

Comment:
What would you suggest as a better error message? How about something along the lines of "cannot specify both '{}' and its alias '{}' in the same cycler"?

One concern I have is that this function can mutate a given cycler, and then raise an exception, which is not optimal.